app/views/events/_event.html.erb

```diff
@@ -0,0 +1,44 @@
+<div class="event-preview <%= 'event-past' if event.is_past %> clearfix">
+  <%= image_tag "example.png", class: "img-rounded pull-right" %>
+  <div class="event-date pull-left">
+    <span class="event-day"><%= l event.start_date, format: "%d" %></span>
+    <span class="event-month"><%= l event.start_date, format: "%b" %></span>
+    <% if event.start_date.year != Time.now.year %>
+      <span class="event-year"><%= event.start_date.year %></span>
+    <% end %>
+  </div>
+
+  <h3>
+    <%= event.name %>
+    <small>
+      <%= tag_label t('activerecord.attributes.event.draft'), :default if event.draft %>
+      <%= tag_label t('events.notices.past_event'), :default if event.is_past %>
+    </small>
+  </h3>
+
+  <p><%= truncate(event.description, length: Event::TRUNCATE_DESCRIPTION_TEXT_LENGTH) do
+    link_to " " + I18n.t('events.list.read_on'), event_path(event)
+  end %></p>
+
+  <%= event.application_deadline_label.try { |label| tag_label label, :default } %>
+  <%= event.duration_label.try { |label| tag_label label, :default } %>
+  <br>
```

Comment:
In my opinion it is always better to use margins instead of hardcoded line breaks especially when it comes to multi device styles.